### ISSUE TITLE
Error reading in Greek text

### ISSUE THREAD

**Issue by jsterchele** [calvinmetcalf/leaflet.shapefile#26]
Hello, I am currently working on a website that maps a shapefile that contains text that is written in Greek.   When I open this shapefile in ArcGIS, the Greek text appears as it should. When I use your plugin on leaflet the text does not render as it should. Here is the image: 
![image](https://cloud.githubusercontent.com/assets/13526588/8915283/040a968c-3472-11e5-91cc-ffe37bcd1323.png)  ONOMA, PERIODOS & XRHSH contain Greek text, but are not rendered as Greek.  Do you have any  ideas to why this could be?

**Comment by calvinmetcalf** [comment]
yeah encoding issues are a wellknown issue with shapefiles/this plugin

**Comment by jsterchele** [comment]
Do you know anyway around this?

**Comment by calvinmetcalf** [comment]
export to geojson you can use something like https://github.com/project-open-data/esri2open to export from arcgis

**Comment by Charmatzis** [comment]
Has your shp file been referenced as UTF-8? 
A quick work around with QGIS is with this plugin 
[Shapefile Encoding Fixer](https://plugins.qgis.org/plugins/shapefile_encoding_fixer/)
